illumio/pce.py

```diff
@@ -503,7 +503,10 @@ def get(self, policy_version: str = DRAFT, parent: Union[str, Reference, dict] =
             """
             endpoint = self._build_endpoint(policy_version, parent)
             response = self.pce.get(endpoint, **{**kwargs, **{'include_org': False}})
-            return [self.object_cls.from_json(o) for o in response.json()]
+            if islist(type(response.json())):
+                return [self.object_cls.from_json(o) for o in response.json()]
+            elif type(response.json()) is dict:
+                return self.object_cls.from_json(response.json())
```

Comment:
Please add another return after the conditional check here - if the value is not a list or a dict we'd currently return `None`. We should instead probably just return `response.json()`.